lib/src/base-styles/_nx-grid.scss

```diff
@@ -39,7 +39,7 @@
 
 .nx-grid-h-keyline {
   border: none;
-  border-top: var(--nx-border-subsection);
+  border-top: var(--nx-border-default);
```

Comment:
I'll have to assume that changing these from secondary to primary was stipulated by @dturner32 , I have no direct way of knowing whether they're correct or not.

Reply:
> I'll have to assume that changing these from secondary to primary was stipulated by @dturner32 , I have no direct way of knowing whether they're correct or not.

Yes, we chatted via Slack